docs/portal/api/addEvent.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: 'addEvent'
+sidebar_position: 1
+---
+# addEvent
+
+* adds an event to the database by adding a document of the specified form to the 'ScheduledEvents' collection of the database (goes through [https://dev.tamudatathon.com/auth/user](https://dev.tamudatathon.com/auth/user) (TODO confirm endpoint) and has the structure:
```

Comment:
The database is a firestore database, and we directly connect to the database after confirming that a valid user is logged in. The authentication details don't need to be mentioned in this file, it should be done in the md file for the `middleware.ts` file.